### DISCUSSION TITLE
Pls unban from discord

### DISCUSSION THREAD

**Discussion by Billcyferka** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1060] [Q&A]
I need depleting reactor designs for RBMK and I gotta join the discord so please unban me

I was banned 3 years ago for posting images of plastic barbie legs
I understand my mistake
Please unban :)

User and tag Billcyferka999#9331

**Comment by Billcyferka** [comment]
@HbmMods

**Reply by RagexPrince683** [reply]
Just make an alt, the community is dogshit

**Reply by LePeeperSauvage** [reply]
Try us, Ragex.

**Comment by RagexPrince683** [comment]
> Try us, Ragex.
I don't need to I live in your head rent free
